edbee-lib/edbee/data/factorykeymap.cpp

```diff
@@ -25,7 +25,9 @@ void FactoryKeyMap::fill( TextEditorKeyMap* km )
     add( "goto_next_word", "move_to_next_word" );
     add( "goto_prev_word", "move_to_previous_word" );
     add( "goto_bol", "move_to_start_of_line" );
+    add( "goto_bol", "move_to_start_of_block");
```

Comment:
Not sure what effect this and the change two lines down do - but is it correct for the two new entries to have the same keys as the adjacent ones - or is it that what ever a block is this is treating it the same as a line? :worried:

Reply:
I think you missed https://github.com/Mudlet/edbee-lib/pull/9, have a look there for the information.